### ISSUE TITLE
NERDTreeFind Not an editor command: NERDTreeFind

### ISSUE THREAD

**Issue by oscarmorrison** [scrooloose/nerdtree#632]
I am having issues getting the NERDTreeFind command to work.

If i have an open NerdTreeBuffer the command works.
But not with a fresh find file.

**Comment by PhilRunninger** [comment]
@oscarmorrison, How do manage your plugins? I'm using **Plug.vim**, and I was able to duplicate this issue only after I made this change:

```
call plug#begin($VIMHOME.'/bundle')
  Plug 'scrooloose/nerdtree', { 'on':  'NERDTreeToggle' }
call plug#end()
```

The second argument to the Plug command loads NERDTree when I toggle it, but if I do `:NERDTreeFind` first, it fails because NERDTree hasn't been loaded yet.

I don't remember if other plugin managers offer the same functionality. If this is not the case for you, provide more information about your setup, so it can be diagnosed.
